node/silkworm/stagedsync/stage_execution.cpp

```diff
@@ -197,24 +198,24 @@ static void revert_state(Bytes key, Bytes value, mdbx::cursor& plain_state_table
         plain_state_table.erase();
     }
     if (!value.empty()) {
-        auto data{location.append(value)};
+        Bytes data{location};
+        data.append(value);
         plain_state_table.upsert(db::to_slice(key1), db::to_slice(data));
     }
 }
 
 // For given changeset cursor/bucket it reverts the changes on states buckets
 static void unwind_state_from_changeset(mdbx::cursor& source, mdbx::cursor& plain_state_table,
                                         mdbx::cursor& plain_code_table, uint64_t unwind_to) {
-    uint64_t block_number{0};
     auto src_data{source.to_last(/*throw_notfound*/ false)};
     while (src_data) {
         Bytes key(db::from_slice(src_data.key));
         Bytes value(db::from_slice(src_data.value));
-        block_number = endian::load_big_u64(&key[0]);
+        const uint64_t block_number = endian::load_big_u64(&key[0]);
```

Comment:
For block numbers we should consistently use the proper alias

```suggestion
        const BlockNum block_number = endian::load_big_u64(&key[0]);
```

Reply:
Right, done.